components/story-inquiry-page/story-concept-faq/StoryConceptFaq.js

```diff
@@ -0,0 +1,34 @@
+import { H1, H2, H3, P } from '../../../style/typography';
+import { getString } from '../../../lib/richText';
```

Comment:
not sure where the onclick is to show a faq, but for mobile is it possible for when you click on one & it shows, and if you click the question again it closes?